crates/ar-cli/src/db.rs

```diff
@@ -0,0 +1,151 @@
+use core::fmt;
+use std::path::PathBuf;
+
+use rusqlite::{Connection, Result, Row};
+
+pub fn init(path: &PathBuf) -> Result<Connection, rusqlite::Error> {
+    Connection::open(path)
+}
+
+#[derive(Debug)]
+pub struct Event {
+    pub tx_hash: Vec<u8>,
```

Comment:
`TxHash` from alloy? similarly `BlockNumber`

Reply:
this is on purpose - I wanted a very basic struct here, that can be passed around